File: program/print-inverted-pyramid-pattern/PrintInvertedPyramidPattern.rb
puts "Enter the inverted pyramid number to display: "
user_input = gets.to_i
def printInvertedPyramidPattern number
    (1..number).each { |i| print "#{i} "}
    puts
    printInvertedPyramidPattern(number - 1) if number > 0 
end

printInvertedPyramidPattern user_input

File: program/print-inverted-pyramid-pattern/print-inverted-pyramid-pattern.py
# print inverted pyramid pattern


def printInvertedPyramidPattern(number: int):
    endNumber = number

    for i in range(0, number):
        for j in range(1, endNumber + 1):
            print(j, end=" ")

        print("")
        endNumber = endNumber - 1


if __name__ == "__main__":
    number = int(input("type a number:"))
    printInvertedPyramidPattern(number)
